BankManagment/BankManagment_Services/AccountTypeService.cs

```diff
@@ -0,0 +1,58 @@
+﻿using BankManagment_Domain.Entity;
+using BankManagment_Infrastructure.Repository;
+using BankManagment_Infrastructure.UnitOfWork;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Services
+{
+    public class AccountTypeService : IAccountTypeService
+    {
+        private readonly IRepository<AccountType> _accountTypeRepository;
+        private readonly IUnitOfWork _unitOfWork;
+
+        public AccountTypeService(IUnitOfWork unitOfWork, IRepository<AccountType> accountTypeRepository)
+        {
+            _unitOfWork = unitOfWork;
+            _accountTypeRepository = accountTypeRepository;
+        }
+
+        public async Task<IEnumerable<AccountType>> GetAllAccountTypesAsync()
+        {
+            return await _accountTypeRepository.GetAllAsync();
```

Comment:
Instead of awaiting here directly return Task<IEnumberable<AccountType>>
Fix in rest of the methods as well